sopel_modules/twitter/twitter.py

```diff
@@ -44,10 +44,14 @@ def get_url(bot, trigger, match):
                      response['status'], match.group(0))
 
     content = json.loads(content.decode('utf-8'))
-    message = ('[Twitter] {content[text]} | {content[user][name]} '
-               '(@{content[user][screen_name]}) | {content[retweet_count]} RTs '
-               '| {content[favorite_count]} ♥s').format(content=content)
-    all_urls = content['entities']['urls']
+    try:
+        message = ('[Twitter] {content[text]} | {content[user][name]} '
+                   '(@{content[user][screen_name]}) | {content[retweet_count]} RTs '
+                   '| {content[favorite_count]} ♥s').format(content=content)
+        all_urls = content['entities']['urls']
+    except:
+        bot.say('[Twitter] Error')
```

Comment:
Awfully terse, isn't it? Maybe "Error fetching tweet. Perhaps it was deleted or made private."